phpseclib/Crypt/Rijndael.php

```diff
@@ -190,29 +190,6 @@ class Rijndael extends Base
     var $kl;
 
     /**
-     * Default Constructor.
```

Comment:
But it is broken here: https://github.com/GrahamForks/phpseclib/blob/d9e9504fbacc20502e4b456891b6f310e8022e91/phpseclib/Crypt/Rijndael.php.